test/FranchiserFactory.invariants.t.sol

```diff
@@ -16,14 +16,31 @@ contract FranchiseFactoryInvariantTest is Test {
         token = new VotingTokenConcrete();
         factory = new FranchiserFactory(IVotingToken(address(token)));
         handler = new FranchiserFactoryHandler(factory);
-        bytes4[] memory selectors = new bytes4[](2);
-        selectors[0] = FranchiserFactoryHandler.handler_fund.selector;
-        selectors[1] = FranchiserFactoryHandler.handler_recall.selector;
+        bytes4[] memory selectors = new bytes4[](6);
+        selectors[0] = FranchiserFactoryHandler.handler_fundMany.selector;
+        selectors[1] = FranchiserFactoryHandler.handler_recallMany.selector;
+        selectors[2] = FranchiserFactoryHandler.handler_recall.selector;
+        selectors[3] = FranchiserFactoryHandler.handler_fund.selector;
+        selectors[4] = FranchiserFactoryHandler.handler_permitAndFund.selector;
+        selectors[5] = FranchiserFactoryHandler.handler_permitAndFundMany.selector;
         targetSelector(FuzzSelector({addr: address(handler), selectors: selectors}));
         targetContract(address(handler));
     }
 
     function invariant_Franchiser_code_size_non_zero() external view {
         assertGt(address(handler.franchiser()).code.length, 0);
     }
+
+    function invariant_FranchiserAndDeployersRecalledBalanceSumMatchesTotalSupply() external view {
```

Comment:
```suggestion
    function invariant_Franchiser_and_deployers_recalled_balance_sum_matches_total_supply() external view {
```

Reply:
Done, and took deployer out of name: f6d0fd2816c6d3b5c46dc2b0a90b1da4b0559356